merino/curated_recommendations/provider.py

```diff
@@ -345,6 +350,28 @@ async def fetch(
 
         return response
 
+    async def fetch_backup_recommendations(
+        self, surface_id: ScheduledSurfaceId
+    ) -> list[CuratedRecommendation]:
+        """Return recommended stories for yesterday's date
+        for a given New Tab surface
+
+        @param: surface_id: a ScheduledSurfaceId
+        @return: A re-ranked list of curated recommendations
```

Comment:
```suggestion
        @return: A list of yesterdays curated recommendations
```
There's no ranking happening here.